ui/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "scality-ui-template",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.7.2",
+    "@kubernetes/client-node": "git://github.com/scality/kubernetes-client-javascript.git#browser",
+    "axios": "^0.18.0",
```

Comment:
`kubernetes-client-javascript` uses `fetch`. Do we really need `axios`?

Reply:
https://medium.com/@thejasonfile/fetch-vs-axios-js-for-making-http-requests-2b261cdd3af5

That's is why axios is part of our boilerplate.

Reply:
I disagree with that article's second part, which claims that a HTTP 400 response is an 'error': no, it's not. The request was handled just fine (so successful), it's just that the request payload denotes an application-level error which you can easily handle using the `fetch` API.

To decode JSON, it's just a function-call away, and when using `async`/`await` it's literally one line of code. Also, a library behind the scenes decoding JSON is icky IMHO.

Anyway, if you want to stick with Axios, fine. I just doubt it's worth the extra dep.